javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -67,21 +67,27 @@ public static void clear() {
      * so they can be compared with ==.
      */
     private static final Map<AnnotationMirror, /*@Interned*/ String> annotationMirrorNames =
-            CollectionUtils.createLRUCache(ANNOTATION_CACHE_SIZE);
+            Collections.synchronizedMap(
```

Comment:
Are you sure?  It's only duplicated twice (the other two cases wrap a `HashMap` rather than an LRU cache).